### DISCUSSION TITLE
Using DateTimePicker within livewire component

### DISCUSSION THREAD

**Discussion by zxtanzxzx** [wireui/wireui#644] [Q&A] [answered]
Below is the code where i am trying to use the datetime picker within my livewire component.
```
    <div class="flex">
        <x-wire-input class="md:p-2 md:text-xl" label="Course Code" lg placeholder="ACBCD1234" />
        <x-wire-input class="md:p-2 md:text-xl" label="Program/Tut. Grp." lg placeholder="DEF2S1G1" />
        <x-wire-input class="md:p-2 md:text-xl" label="Original Class Venue" lg placeholder="Venue No" />
        <x-wire-datetime-picker label="Origina Class Date Time" placeholder="Original Class Date" wire:model.defer="normalPicker" />
    </div>
```

```
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <title>TEST</title>

    <!-- Styles -->
    {{-- @vite(['resources/css/app.css']) --}}
    @vite(['resources/css/app.css', 'resources/js/app.js'])

    @wireUiScripts
    <script src="//unpkg.com/alpinejs" defer></script>
</head>
```
The date time picker shows a undefined and did not work
![image](https://user-images.githubusercontent.com/12552846/232182874-a5e891d2-a9a2-4eac-810e-b87dad1b62bf.png)

Consoles showing  a bunch of erros and warnings
![image](https://user-images.githubusercontent.com/12552846/232183291-9f58c03c-e401-41ab-a456-68bb3b390f36.png)

Tried put the datetime picker directly on the blade, work well, undefined shows when its in a livewire component

How can i make it works, Thanks !

**Comment by geoffreyvanwyk** [comment, marked as answer]
Place `@wireUiScripts` above `vite...`.
